src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -685,11 +685,17 @@ - (void)_setDSTOffsets:(NSArray<MTRTimeSynchronizationClusterDSTOffsetStruct *>
 #define MTR_DEVICE_SUBSCRIPTION_MAX_INTERVAL_MIN (10 * 60) // 10 minutes (for now)
 #define MTR_DEVICE_SUBSCRIPTION_MAX_INTERVAL_MAX (60 * 60) // 60 minutes
 
+- (BOOL)_subscriptionsAllowed
+{
+    return ![_deviceController isKindOfClass:MTRDeviceControllerOverXPC.class];
```

Comment:
Shouldn't we use this helper in the other place where we are doing this check?

Reply:
done.